src/main/lib/icon/IconFileStorage.js

```diff
@@ -37,25 +39,6 @@ export default class IconFileStorage extends StorageInterface {
     return fs.unlink(this._buildKeyPath(iconKey));
   }
 
-  /**
-     * Encode icon data for storage
-     * @param {Buffer} data The data buffer for encoding
-     * @returns {Promise.<*>} A promise that resolves with the encoded data ready
-     *  for storage
-     */
-  encodeIconForStorage(data) {
-    return Promise.resolve(data);
-  }
-
-  /**
-     * Fetch all icon keys
-     * @returns {Promise.<Array.<String>>} A promise that resolves with an array of icon keys
-     */
-  async getIconKeys() {
-    const path = this.path;
-    return this._retryWithDirCreation(() => fs.readdir(path));
-  }
```

Comment:
I get the point that this is not used now in `Iconographer`, but maybe this means that this method should not be in the interface either?